src/ir_Pioneer.cpp

```diff
@@ -30,13 +30,19 @@
 //  http://adrian-kingston.com/IRFormatPioneer.htm
 void IRsend::sendPioneer(const uint64_t data, const uint16_t nbits,
                          const uint16_t repeat) {
-  // If nbits is to big, or is odd, abort.
-  if (nbits > sizeof(data) * 8 || nbits % 2 == 1) return;
```

Comment:
`if (nbits > sizeof(data) * 8) return;`
We probably shouldn't end more than 64 bits. (i.e. data is a `uint64_t`)

Reply:
Reverted/fixed